librz/core/cfile.c

```diff
@@ -925,6 +925,7 @@ RZ_API bool rz_core_bin_load(RZ_NONNULL RzCore *r, RZ_NULLABLE const char *filen
 	bool is_io_load = false;
 	const char *cmd_load;
 	if (!cf) {
+		eprintf("NOPE\n");
```

Comment:
ORLY?
Also `RZ_LOG_ERROR()`

Reply:
i think i will switch that check to rz_return_val_if_fail